### PR TITLE
Feat/account info

### PR DESCRIPTION
**Pull request by joshLong145**
addition of account info command listing
Aliases
Account address
current token

## Changes

- Item 1
- Item 2

## Checklist

- [ ] Unit tests
- [ ] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: cmd/commands/registry/info.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+				return
+			}
+
+			sessionInfo, err := cli.GetSessionInfo()
+
+			if err != nil {
+				logger.Fatalf("Error while getting session info: %s\n", err.Error())
+			}
+
+			logger.Printf("├── Current address: %s", sessionInfo.Info.Address)
```
Comment (mcjcloud) [inline cmd/commands/registry/info.go:30]: Missing newlines in these

**Review by ntindle — APPROVED** [review]

**Review by 6missedcalls — APPROVED** [review]
LGTM

**Comment by ntindle** [comment]
Don't merge this unless absolutely critical. Coming up on the code freeze time

**Comment by ntindle** [comment]
But good feature

**Comment by mcjcloud** [comment]
@ntindle @joshLong145 Maybe we change this to target `dev` so it can be merged in now?

**Pull request merged**